salt/common/tools/sbin/so-status

```diff
@@ -290,22 +285,31 @@ main() {
 
 # {% endraw %}
 
+is_tty() {
+    __tty=0
+    [ -t 1 ] && __tty=1
+    # don't print colors if NO_COLOR is set to anything
+    [ "${#NO_COLOR}" -ne 0 ] && __tty=0
+}
+
+if ! [ "$(id -u)" = 0 ]; then
+   echo "${0}: This command must be run as root"
+fi
```

Comment:
This should also exit the script with a nonzero exit code.

Reply:
done